frontend/src/pages/home/Home.tsx

```diff
@@ -13,7 +13,8 @@ import LandingPageHomeHint from './LandingPageHomeHint';
 
 const Home: React.FC = () => {
   const { status: projectsAvailable } = useIsAreaAvailable(SupportedArea.DS_PROJECTS_VIEW);
-  const { status: modelCatalogAvailable } = useIsAreaAvailable(SupportedArea.MODEL_CATALOG);
+  // TODO: Temporarily disabled model catalog section - to be re-enabled in future
+  // const { status: modelCatalogAvailable } = useIsAreaAvailable(SupportedArea.MODEL_CATALOG);
```

Comment:
Let's link to this issue in this comment: https://issues.redhat.com/browse/RHOAIENG-34405